ciao-controller/main.go

```diff
@@ -61,6 +68,7 @@ var servicePassword = ""
 var volumeAPIPort = block.APIPort
 var computeAPIPort = compute.APIPort
 var imageAPIPort = osimage.APIPort
+var controllerAPIPort = api.Port
```

Comment:
it will also require some configuration changes since we'll want this passed with the config from scheduler.